src/_base/docker/image/php-fpm/Dockerfile.twig

```
@@ -9,13 +9,15 @@ FROM {{ @('docker.image.php-fpm') }}
 WORKDIR /app
 COPY root  /
 
-RUN curl --fail --silent --location --output /sbin/tini https://github.com/krallin/tini/releases/download/v0.19.0/tini \
+# installing tools in the image is deprecated
+RUN ([ -e /sbin/tini ] || curl --fail --silent --location --output /sbin/tini https://github.com/krallin/tini/releases/download/v0.19.0/tini) \
  && chmod +x /sbin/tini \
- && curl --fail --silent --location --output /usr/local/bin/mhsendmail https://github.com/mailhog/mhsendmail/releases/download/v0.2.0/mhsendmail_linux_amd64 \
+ && ([ -e /usr/local/bin/mhsendmail ] || curl --fail --silent --location --output /usr/local/bin/mhsendmail https://github.com/mailhog/mhsendmail/releases/download/v0.2.0/mhsendmail_linux_amd64) \
  && chmod +x /usr/local/bin/mhsendmail
+
 {%- set install_extensions=@('php.install_extensions')|merge(@('php.fpm.install_extensions'))|filter(v => v is not empty) %}
-{%- if install_extensions %} \
- && cd /root/installer \
+{%- if install_extensions %}
```

```suggestion
{%- if install_extensions %} \
```
Should fix the build error - this gets put on the same line as `chmod +x /usr/local/bin/mhsendmail` by twig.

Perhaps there's a better way to do it?

---

The better way is to not use - in twig as it already does trim a bit anyway. I've removed most of them used there, and added one. makes xdebug the line after extensions rather than having 2 lines gap if <3, and still has an empty line above it if no extensions